packages/web-app-files/tests/unit/views/spaces/GenericSpace.spec.ts

```diff
@@ -312,7 +317,7 @@ function getMountedWrapper({
     {
       id: 'com.github.owncloud.web.files.folder-view.resource-table',
       type: 'folderView',
-      scopes: ['resource', 'space', 'favorite'],
+      extensionPointIds: [folderViewsFolder, folderViewsProjectSpaces, folderViewsFavorites],
```

Comment:
I'm wondering where to draw the line between a new extension point id and categories